platform_versioned_docs/version-24.2/monitoring/configtables/log_events.yml

```diff
@@ -15,6 +15,14 @@
   Resource:            'Data Explorer cloud buckets'
   Events logged:       'Add, edit, delete, hide, show'
   Note:                'Events for public and private buckets are logged.'
+-
+  Resource: 'Data Studios sessions'
+  Events logged: 'Add, started, connected, disconnected, stopped, deleted'
```

Comment:
```suggestion
  Events logged: 'Add, start, connect, disconnect, stop, delete'
```

Reply:
None of the other audit log entry verbs are past tense - they are all present tense.